src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adjourncase/AdjournCasePreviewService.java

```diff
@@ -179,9 +185,14 @@ private void handleFaceToFaceHearing(Adjournment adjournment, AdjournCaseTemplat
     private void setIntepreterDescriptionIfRequired(AdjournCaseTemplateBodyBuilder adjournCaseBuilder, SscsCaseData caseData) {
         if (isYes(caseData.getAdjournment().getInterpreterRequired())) {
             if (caseData.getAdjournment().getInterpreterLanguage() != null) {
-                adjournCaseBuilder.interpreterDescription(
-                    languageService.getInterpreterDescriptionForLanguageKey(
-                        caseData.getAdjournment().getInterpreterLanguage()));
+                String languageLabel = caseData.getAdjournment().getInterpreterLanguage().getValue().getLabel();
+                String languageKey = caseData.getAdjournment().getInterpreterLanguage().getValue().getCode();
+                Language language = signLanguagesService.getLanguageByHmcReference(languageKey);
```

Comment:
This is a bit confusing naming the variable 'language' when it is retrieved from signLanguageService.

Reply:
renamed